compiler/syntax/src/res_outcome_printer.ml

```diff
@@ -123,19 +123,6 @@ let rec print_out_type_doc (out_type : Outcometree.out_type) =
            (if non_gen then Doc.text "_" else Doc.nil);
            Doc.lbracket;
            Doc.indent (Doc.concat [opening; print_out_variant out_variant]);
-           (match labels with
```

Comment:
Are there any cases when we do want to print out the variant's labels/tags?

The code just above mentions syntax like `[< #A | #B > #X #Y ]`, but this doesn't seem to be valid ReScript:

https://github.com/rescript-lang/rescript/blob/cfcd99b3af96aa578dd5a0c767384ef19ba58e7b/compiler/syntax/src/res_outcome_printer.ml#L104-L116